cypress/integration/tests/developer/applicationinventory/analysis/binary_analysis.test.ts

```diff
@@ -31,7 +31,7 @@ import { Analysis } from "../../../../models/developer/applicationinventory/anal
 import { CredentialType, UserCredentials } from "../../../../types/constants";
 import { CredentialsSourceControlUsername } from "../../../../models/administrator/credentials/credentialsSourceControlUsername";
 import { CredentialsMaven } from "../../../../models/administrator/credentials/credentialsMaven";
-import { Proxy } from "../../../../models/administrator/proxy/proxy";
+// import { Proxy } from "../../../../models/administrator/proxy/proxy";
```

Comment:
Why is this commented ? 
If not needed , please remove .

Reply:
Fixed